pytorch_to_returnn/torch/nn/modules/shape.py

```diff
@@ -404,15 +402,6 @@ def _convert_dim_returnn(x: Union[SizeValue, int, Tensor]) -> Union[int, Dim]:
   raise TypeError(f"Convert dim to RETURNN: cannot handle dim {x!r} of type {type(x)}")
 
 
-def _convert_dim_torch(x: Union[SizeValue, int, Tensor]) -> Union[int, SizeValue]:
```

Comment:
Why did you move this?

Reply:
Because it was not used in the other file anymore and I found it awkward to have it defined in a file where it's not used while it's used in only one other file.